integrations/symfony-bundle.rst

```diff
@@ -56,6 +56,44 @@ Web Debug Toolbar
 
 When using a client configured with ``HttplugBundle``, you will get debug information in the web debug toolbar. It will tell you how many request were made and how many of those that were successful or not. It will also show you detailed information about each request.
```

Comment:
the logging is also controlled by a config setting that defaults to kernel.debug, right? we could maybe mention that below, near where we talk about the body capturing configuration